### PR TITLE
doc: remove missing js file.

### PR DESCRIPTION
**Pull request by joelin**
the js file http://ayni.ceph.com/public/js/ceph.js is not avaliabled.

Signed-off-by: linxuhua <linxuhua@unionpay.com>

### CODE REVIEW

**Comment by tchaikov** [comment]
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes, in this case, it would be "doc: ".

**Comment by tchaikov** [comment]
see also https://www.spinics.net/lists/ceph-devel/msg36405.html

**Comment by joelin** [comment]
@tchaikov  ok

File: doc/_templates/layout.html [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {% extends "!layout.html" %}
 
 {%- block extrahead %}
-    <script type="text/javascript" src="http://ayni.ceph.com/public/js/ceph.js"></script>
+   <!--  <script type="text/javascript"
```
Comment (tchaikov) [inline doc/_templates/layout.html]: could just remove file. in other words, to revert 35663fa55ac1579a3b0c8b67028a3a8dfea87b48. but let's wait for @alfredodeza 's call.

**Comment by alfredodeza** [comment]
@djgalloway this is part of the "multi-version" doc support as referenced in http://tracker.ceph.com/issues/18075

Do you think we can address this? I do think we need something to navigate different versions.

@joelin The problem with removing this today is that when we add it back, the gap in docs will mean some docs will not have the menu available.

**Comment by djgalloway** [comment]
@alfredodeza TBH I think the overlay/bar that ayni provided was a little obtrusive and didn't mesh well with the overall look and feel of the docs.ceph.com layout.

What about adding something to the sidebar that indicates which version of the docs are being displayed.  Then users can click on the version number that takes them to another page with a list of available doc versions?

**Comment by alfredodeza** [comment]
@djgalloway I don't think any styling is set in stone. That can surely be improved (even though that is how every hosted Sphinx docs is on readthedocs.org). There are two problems right now:

* there is nothing today to help navigate different versions
* this PR would remove the file creating a gap that can't be fixed unless backported

**Comment by joelin** [comment]
@alfredodeza this is my confusion. i am working in two version now. but i can't got the old docs in website. so i build it in intranet.  maybe i can working for this.  I will submit a PR in the next few days. can anyone give me the structure of folder for docs as below?
muti-version

**Comment by djgalloway** [comment]
@joelin Our directory structure at that level is the release name that is a symlink back to the sha1 of the tip of the branch.  e.g.,

```
$ ls -lah /var/docs/luminous
lrwxrwxrwx 1 XXXXX XXXXX 48 Aug 30 20:46 luminous -> ../sha1/fa996769fcf1d1eb135c00d98b7b48e370f210d0

$ ls /var/docs/luminous/
api           cephfs       dev         genindex  _images     install  mgr          py-modindex  radosgw  release-notes  search          sha1      start
architecture  ceph-volume  _downloads  glossary  index.html  man      objects.inv  rados        rbd      releases       searchindex.js  _sources  _static
```

The webserver, nginx, has a bunch of location definitions that serve up the proper directory based on release name (or branch) in the URL.

```
    location ~ ^/docs/(.*) {
        alias /var/docs/$1;
    }
```

Does that answer your question?

**Comment by tchaikov** [comment]
@alfredodeza is this change still relevant ? if we are expecting a better solution, probably we should close this PR at this moment.

@noahdesu FWIW, since "Version navigation and alerting" is listed in https://pad.ceph.com/p/Ceph_Documentation. i am connecting the pad document to http://tracker.ceph.com/issues/18075

**Comment by alfredodeza** [comment]
I think I would like to defer to @noahdesu and @neha-ojha 

I'm fine if they want to close this (or even merge)

**Comment by noahdesu** [comment]
My instinct is to close this PR. But if it is solving any problem at all for anyone, merge it. Adding the version switching to the docs is on our roadmap, and will probably happen in the next phase. That will necessarily mean examining this existing javascript solution and making the keep/toss decision.

**Pull request closed without merging**